v63002/u_inref/gtm8281.csh

```diff
@@ -0,0 +1,37 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+# Tests that the new maximum for source lines, xecute strings and direct mode input all accept up to 8192 bytes
+#
+
+# 1 terminating characters and 9 other characters outside of the big string
+# Expect the line to be too long as a source line when n=8182, and too long as
+# an xecute string or command in direct mode when n=8183
+foreach n (8180 8181 8182 8283 8184)
```

Comment:
Please process the output by replacing an 8180-byte-string with say "a <repeated 8180 times>". That way the reference file will be readable instead of containing HUGE strings.